src/Ui.Altair/Internal/AltairPageModel.cs

```diff
@@ -37,11 +36,20 @@ public string Render()
             var builder = new StringBuilder(streamReader.ReadToEnd())
                 .Replace("@Model.GraphQLEndPoint", _options.GraphQLEndPoint)
                 .Replace("@Model.SubscriptionsEndPoint", _options.SubscriptionsEndPoint)
-                .Replace("@Model.Headers", JsonSerializer.Serialize<object>(headers));
+                .Replace("@Model.Headers", JsonSerialize(headers));
 
             _altairCSHtml = _options.PostConfigure(_options, builder.ToString());
         }
 
         return _altairCSHtml;
     }
+
+    private static string JsonSerialize(object value)
+    {
+#if !NETSTANDARD2_0
```

Comment:
I would invert condition.

Reply:
Ok

Reply:
Btw, you have requested this change many times, and I do not agree with the general principle that conditions should be inverted if they have a `!`.  In this instance the ifdef would have matched the other ifdef in the project, and would place the typical/default/primary code first.  I think you should reconsider your viewpoint.  Regardless, I changed it as you requested.

Reply:
I reason from the simplicity of reading code when you do not mentally invert condition even if typical/default/primary code implies `!`.